Makefile

```diff
@@ -0,0 +1,46 @@
+# Definition of variables
+POETRY := poetry
+
+
+# Basic commands
+install:
+	@echo "Installing project dependencies..."
+	$(POETRY) install
+
+
+setup:
```

Comment:
How we can setup via make, if make is a part of repository, lets remove that command